<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>MiSans</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPlainTextEdit" name="RecieveEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>461</width>
     <height>221</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="SendEdit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>280</y>
     <width>241</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>481</width>
     <height>251</height>
    </rect>
   </property>
   <property name="title">
    <string>接收框</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LocalPort">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="TargetIP">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>290</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>本地端口</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>340</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>目标端口</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="TargetPort">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>390</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>目标IP</string>
   </property>
  </widget>
  <widget class="QPushButton" name="OpenButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>440</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CloseButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>440</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SendButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>440</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>260</y>
     <width>261</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>发送框</string>
   </property>
  </widget>
  <zorder>groupBox_2</zorder>
  <zorder>groupBox</zorder>
  <zorder>RecieveEdit</zorder>
  <zorder>SendEdit</zorder>
  <zorder>LocalPort</zorder>
  <zorder>TargetIP</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>TargetPort</zorder>
  <zorder>label_3</zorder>
  <zorder>OpenButton</zorder>
  <zorder>CloseButton</zorder>
  <zorder>SendButton</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
